util/test/checkChplInstall

```diff
@@ -25,16 +25,35 @@
 # To use something other than $HOME/.chpl, set the CHPL_CHECK_INSTALL_DIR
 # environment variable.
 
+function log_info()
+{
+    local msg=$@
+    echo "[INFO] ${msg}"
+}
 
-# Base name for test job in $CHPL_HOME/test/release/examples
-TEST_JOB_BASENAME=hello6-taskpar-dist
+function log_warning()
+{
+    local msg=$@
+    echo "[WARNING] ${msg}"
+}
 
-err() {
-    echo "error: $@" >&2
+function log_error()
+{
+    local msg=$@
+    echo "[ERROR] ${msg}" >&2
 }
 
+function log_success()
+{
+    local msg=$@
+    echo "[SUCCESS] ${msg}"
```

Comment:
I'm happy with it as is  - maybe it is better  to have different log messages stand out from each other. We can always change it in the future if need be.